src/documents/timecard.md

```diff
@@ -0,0 +1,55 @@
+# Timecard
+
+
+The timecard page enables the user to record their work times via a sequence of screens. Currently the only developed timecardPeriodType is Shift. After choosing this radio button option, the user is taken to the TimecardEntriesList component where they can input the times and dates worked:
+
+![Alt text](images/Screenshot%202023-06-22%20at%2008.15.32.png)
+
+
+After clicking save, the screen displays the recorded shift with the option to remove or edit it: 
+
+![Alt text](images/Screenshot%202023-06-22%20at%2008.46.42.png)
+
+When editing the saved time, the function 'isFinishTimeOnNextDay' calculates whether the time is on the next day and the UI notifies the user with dynamic text:
+
+![Alt text](images/Screenshot%202023-06-22%20at%2009.26.14.png)
+
+The date input is conditionally rendered by clicking the checkbox. The dates are automatically inserted. If 'isFinishTimeOnNextDay' is true, the finish date is set to the next day. These can be manually changed: 
+
+![Alt text](images/Screenshot%202023-06-22%20at%2010.14.50.png)
+
+The user is returned to the the Timecard of the start date:
```

Comment:
The user is never re-routed, they will remain on the day they were originally viewing.